decidim-budgets/app/views/decidim/budgets/projects/_order_progress_summary_content.html.erb

```diff
@@ -20,7 +20,7 @@
       <% end %>
     <% end %>
     <div class="budget-summary__progressbar">
-      <div class="progress budget-progress" role="progressbar" aria-valuenow="<%= current_order_budget_percent %>" aria-valuetext="<%= current_order_budget_percent %>%" aria-valuemin="0" aria-valuemax="100">
+      <div class="progress budget-progress" role="progressbar" aria-label="<%= current_order_budget_percent %>" aria-valuenow="<%= current_order_budget_percent %>" aria-valuetext="<%= current_order_budget_percent %>%" aria-valuemin="0" aria-valuemax="100">
```

Comment:
Sorry but I still noticed the same issue with this progress bar label (`aria-label`) as the other one.

Also, here we are using the percent sign `%` instead of the literal text `percent` as with the other progress bar. I think both are fine but we should be consistent.

Reply:
Done